app/models/concerns/hyrax/collection_nesting.rb

```diff
@@ -49,5 +49,9 @@ def find_children_of(destroyed_id:)
     def use_nested_reindexing?
       true
     end
+
+    def reindex_id(id:)
```

Comment:
Is this method needed as a public method or is it just to DRY up other methods?  If it needs to be public how about renaming it to something more specific like `reindex_nested_relationships_for(id:)` since it will be visible on all models that include this module?

Reply:
I just did this to DRY other methods.